image-search/Models/AlbumGenerator.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  AlbumGenerator.swift
+//  image-search
+//
+//  Created by Renato Vieira on 08/10/19.
+//  Copyright © 2019 Stant. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+func requestAlbum(url: String, view: ViewController) {
+    if let fakeUrl = URL(string: "http://fakerapiexample.herokuapp.com/album") {
+        URLSession.shared.dataTask(with: fakeUrl) { (data, response, error) in
+            guard let data = data else {
+                
+                DispatchQueue.main.async {
+                    Alert().display(view.self, "Attention", error?.localizedDescription ?? "Internet OffLine")
```

Comment:
Here is is better is this alert showing stays in a function of view controller and you just call i like: 

view.nameOfYouFunction()

Something like "showErrorAlert".

Reply:
I changed the function name to showErrorAlert, but I didn't understand this part: view.nameOfYouFunction ()